src/api/clients.coffee

```diff
@@ -6,176 +6,148 @@ authorisation = require './authorisation'
 ###
 # Adds a client
 ###
-exports.addClient = `function *addClient() {
+exports.addClient = () ->
 
-  // Test if the user is authorised
-  if (authorisation.inGroup('admin', this.authenticated) === false) {
-    logger.info('User ' +this.authenticated.email+ ' is not an admin, API access to addClient denied.')
-    this.body = 'User ' +this.authenticated.email+ ' is not an admin, API access to addClient denied.'
-    this.status = 'forbidden';
-    return;
-  }
+  # Test if the user is authorised
+  if not authorisation.inGroup 'admin', this.authenticated
+    logger.info "User #{this.authenticated.email} is not an admin, API access to addClient denied."
+    this.body = "User #{this.authenticated.email} is not an admin, API access to addClient denied."
+    this.status = 'forbidden'
```

Comment:
It might be a good time to switch to Ryan and Martin's `logAndResponse` utility here (https://github.com/jembi/openhim-core-js/blob/convert-javascript/src/api/channels.coffee#L42)

There are a few other places where it could be useful as well.